packages/messenger-widget/src/utils/ens-utils.ts

```diff
@@ -95,14 +95,26 @@ export const onClose = (
     dispatch: React.Dispatch<Actions>,
     setSelectedContact: Function,
     screenWidth: number,
+    showContacts: boolean,
 ) => {
     if (screenWidth && screenWidth > MOBILE_SCREEN_WIDTH) {
```

Comment:
remove else and add screenWidth && screenWidth > MOBILE_SCREEN_WIDTH to the if condition of showcontacts